tweets-service/service.yml

```diff
@@ -1,38 +1,45 @@
-type: exosphere-tweets-service
+type: worker
 description: Generic MongoDB storage service
 author: originate
 
 
 startup:
-  command: node_modules/exoservice/bin/exo-js
   online-text: online at port
 tests: node_modules/.bin/cucumber-js
 
 
-messages:
-  receives:
-    - create tweet
-    - list tweets
-    - get tweet details
-    - update tweet
-    - delete tweet
-  sends:
-    - tweet created
-    - tweets listed
-    - tweet details
-    - tweet not found
-    - tweet updated
-    - tweet deleted
-    - tweet not created
-
-
-dependencies:
-  - name: mongo
-    version: 3.4.0
-
-
-# production:
-#   aws:
-#     cpu: 100
-#     memory: 500
-#     essential: true
+dependency-data:
+  exocom:
+    receives:
+      - create tweet
+      - list tweets
+      - get tweet details
+      - update tweet
+      - delete tweet
+    sends:
+      - tweet created
+      - tweets listed
+      - tweet details
+      - tweet not found
+      - tweet updated
+      - tweet deleted
+      - tweet not created
+
+local:
+  scripts:
+    run: lsc src/server.ls
+    test: /node_modules/.bin/cucumber-js
+  secrets:
+    - MONGODB_USER
+    - MONGODB_PW
+
+remote:
+  cpu: 100
+  memory: 500
+  secrets:
+    - MONGODB_USER
+    - MONGODB_PW
+
+user-data:
```

Comment:
Can we remove this from here and hardcode it in the source? I see no reason it needs to be in here. Same eleswhere